src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -384,7 +384,6 @@ private SourceRepository CreateRepositoryFromSource(string source)
         {
             var packageSource = new PackageSource(source);
             var repository = _sourceRepositoryProvider.CreateRepository(packageSource);
-            var resource = repository.GetResource<PackageSearchResource>();
```

Comment:
do you why was the search resource fetched here? I'm guessing just a bug?

Reply:
It certainly looks like a bug. `resource` was never used, and `GetResource` isn't supposed to have side-effects, so I don't see any reason why the line should exist. Hence I'm deleting it.  We have a fair few PMC tests, so I'm hopeful that if deleting this line does somehow cause problems, the tests should catch it.